include/boost/geometry/strategies/geographic/closest_points_pt_seg.hpp

```diff
@@ -12,8 +14,6 @@
 
 #include <algorithm>
```

Comment:
Is this include possibly unused too?

Reply:
Yes, and several others in this file. They were probably copied from spherical version. I'll remove them.